Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx

```diff
@@ -1477,7 +1477,7 @@ void qSlicerMarkupsModuleWidget::onAddMarkupPushButtonClicked()
     return;
     }
   // get the active node
-  bool definedStatus = d->MarkupsNode->GetMaximumNumberOfControlPoints() != 0;
+  bool definedStatus = d->MarkupsNode->GetMaximumNumberOfControlPoints() >= 0;
```

Comment:
This `definedStatus` name is not relevant here (probably just some copy-paste remnant). Could you change it to the simple check that we do elsewhere?